### PR TITLE
Display timeout flags in help command and doc

### PR DESCRIPTION
**Pull request by jiayiwang7**
*Issue #, if available:*

Related issue: #5637 

*Description of changes:*

Display the timeout flags in CLI help and doc.

*Testing (if applicable):*

```
$ eksctl anywhere create cluster -h

This command is used to create workload clusters

Usage:
  anywhere create cluster -f <cluster-config-file> [flags]

Flags:
      --bundles-override string             Override default Bundles manifest (not recommended)
      --control-plane-wait-timeout string   Override the default control plane wait timeout (default "1h0m0s")
      --external-etcd-wait-timeout string   Override the default external etcd wait timeout (default "1h0m0s")
  -f, --filename string                     Filename that contains EKS-A cluster configuration
      --force-cleanup                       Force deletion of previously created bootstrap cluster
  -z, --hardware-csv string                 Path to a CSV file containing hardware data.
  -h, --help                                help for cluster
      --install-packages string             Location of curated packages configuration files to install to the cluster
      --kubeconfig string                   Management cluster kubeconfig file
      --no-timeouts                         Disable timeout for all wait operations
      --node-startup-timeout string         Override the default node startup timeout (default "10m0s")
      --per-machine-wait-timeout string     Override the default machine wait timeout per machine (default "10m0s")
      --skip-ip-check                       Skip check for whether cluster control plane ip is in use
      --tinkerbell-bootstrap-ip string      Override the local tinkerbell IP in the bootstrap cluster
      --unhealthy-machine-timeout string    Override the default unhealthy machine timeout (default "5m0s")

Global Flags:
  -v, --verbosity int   Set the log level verbosity
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/reference/eksctl/anywhere_upgrade_cluster.md [hunk 1]
```diff
@@ -18,14 +18,19 @@ anywhere upgrade cluster [flags]
...
-      --bundles-override string   Override default Bundles manifest (not recommended)
-  -f, --filename string           Filename that contains EKS-A cluster configuration
-      --force-cleanup             Force deletion of previously created bootstrap cluster
-  -z, --hardware-csv string       Path to a CSV file containing hardware data.
-  -h, --help                      help for cluster
-      --kubeconfig string         Management cluster kubeconfig file
-      --no-timeouts               Disable timeout for all wait operations
-  -w, --w-config string           Kubeconfig file to use when upgrading a workload cluster
+      --bundles-override string             Override default Bundles manifest (not recommended)
+      --control-plane-wait-timeout string   Override the default control plane wait timeout (60m). (default "1h0m0s")
```
Comment (d8660091) [inline docs/content/en/docs/reference/eksctl/anywhere_upgrade_cluster.md:22]: Should we remove this redundant "(60m)"?

**Review by g-gaston — APPROVED** [review]

**Pull request merged**